shaders/graph.glsl

```diff
@@ -13,8 +13,17 @@
 #define DRAW_OUTLINE 0
 /* 1 to draw edge highlight, 0 to disable */
 #define DRAW_HIGHLIGHT 1
+/* Whether to anti-alias the border of the graph, creating a smoother curve.
+   This may have a small impact on performance. Note that this only anti-aliases
```

Comment:
I put in the performance note just to be sure, but I did not really notice any performance drop on my machine, so you might consider leaving it out depending on how it runs for you.